smartapps/smartthings/notify-me-when.src/notify-me-when.groovy

```diff
@@ -111,16 +111,22 @@ private sendMessage(evt) {
 	if (location.contactBookEnabled) {
 		sendNotificationToContacts(msg, recipients, options)
 	} else {
-		if (pushAndPhone != 'No') {
-			log.debug 'sending push'
-			options.method = 'push'
-			sendNotification(msg, options)
-		}
 		if (phone) {
 			options.phone = phone
-			log.debug 'sending SMS'
-			sendNotification(msg, options)
+			if (pushAndPhone != 'No') {
+				log.debug 'Sending push and SMS'
+				options.method = 'both'
+			else {
+				log.debug 'Sending SMS'
+				options.method = 'phone'
+			}
+		else if (pushAndPhone != 'No') {
+			log.debug 'Sending push'
+			options.method = 'push'
+		} else {
+			log.debug 'Sending nothing'
```

Comment:
I think you may have to set `options.method = 'none'` here or you'll still get a push notification